base/system/diskpart/lang/en-US.rc

```diff
@@ -51,7 +51,7 @@ BEGIN
     IDS_LIST_PARTITION_NO_DISK "\nThere is no disk to list partitions.\nPlease select a disk and try again.\n\n"
     IDS_LIST_VOLUME_HEAD "  Volume ###  Ltr  Label        FS     Type        Size     Status   Info\n"
     IDS_LIST_VOLUME_LINE "  ----------  ---  -----------  -----  ----------  -------  -------  --------\n"
-    IDS_LIST_VOLUME_FORMAT "  Volume %3lu   %c   %-11s  %-5s  %10u  %4I64u %-2s\n"
+    IDS_LIST_VOLUME_FORMAT "  Volume %-3lu   %c   %-11.11s  %-5s  %10u  %4I64u %-2s\n"
```

Comment:
Regarding the `%-11.11s` : isn't this equivalent to `%-.11s` ?

Reply:
Yes it is:

![image](https://user-images.githubusercontent.com/578406/165535871-50ad5148-f897-4d1f-b8e7-6aca8f4ec395.png)

Reply:
So yes, this could be used instead for simplification purposes.

Reply:
![image](https://user-images.githubusercontent.com/57249267/165549877-096a7795-f0f4-438a-8362-871a206d4825.png)

Reply:
ok then restore the thing 😅